src/cmd/policy.go

```diff
@@ -21,6 +24,63 @@ type regoInput struct {
 	Files []string `json:"files"`
 }
 
+type gitlabRepoMetadata struct {
+	Name        string             `json:"name,omitempty"`
+	Description string             `json:"description,omitempty"`
+	Language    string             `json:"language,omitempty"`
+	Languages   map[string]float64 `json:"languages,omitempty"`
+}
+
+type githubRepoMetadata struct {
+	Name        string             `json:"name,omitempty"`
+	Description string             `json:"description,omitempty"`
+	Language    string             `json:"language,omitempty"`
+	Languages   map[string]float64 `json:"languages,omitempty"`
+}
+
+type genericStruct interface {
+	gitlabRepoMetadata | githubRepoMetadata | opslevel.Level
+}
+
+func structToValue[S genericStruct](s S) ast.Value {
```

Comment:
I don't know if I did this right but I thought it was really cool either way. lol

Reply:
Ok, awesome! Making those changes now.